src/app/components/shared/website-capability-warning/website-status-warning.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { WebsiteStatusService } from "@baw-api/website-status/website-status.service";
+import { websiteStatusMenuItem } from "@components/website-status/website-status.menu";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { WebsiteStatus } from "@models/WebsiteStatus";
+import { takeUntil } from "rxjs";
+
+@Component({
+  selector: "baw-website-status-warning",
+  template: `
+    <div *ngIf="!isCapabilityHealthy" class="alert alert-danger">
+      The ability to {{ websiteCapabilityText }} is temporarily unavailable.
+      Please try again later.
```

Comment:
This should support a slot. e.g. usage should be 

```html
<baw-website-status-warning>
    The upload functionis currently unavailable.
</baw-website-status-warning>
```

You can keep the attribute too - much in the same way that ngbTooltip can accept either text in an attribute or a slot - but it's always smart to add the slot as an escape hatch in case we need to put some other content in.